spec/support/.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+inherit_from:
+  - ../../.rubocop.yml
```

Comment:
Makes Rubocop check for single quotes.  The files in the support directory are Ruby files, not RSpec, so I use the normal Rubocop rules on those.  Otherwise, the Rubocop config in spec would apply to the support files.